CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@
 
 - [IPs] Handle space-delimited IP address format provided by outdated guest tools [5801](https://github.com/vatesfr/xen-orchestra/issues/5801) (PR [5805](https://github.com/vatesfr/xen-orchestra/pull/5805))
 - [API/pool.listPoolsMatchingCriteria] fix `unknown error from the peer` error (PR [5807](https://github.com/vatesfr/xen-orchestra/pull/5807))
+- [Plugins/perf-alert] Ability to choose only all running hosts and VMs (PR [5811](https://github.com/vatesfr/xen-orchestra/pull/5811))
```

Comment:
This is unclear, maybe something like `*all mode* only selects running hosts and VMs` would be better?